.example.env

```diff
@@ -2,4 +2,7 @@ HOST=host
 PORT=1337
 USERNAME=username
 PASSWORD=password
-DATABASE=dbname
\ No newline at end of file
+DATABASE=dbname
+
+JWTSECRET=somesecretstring
+TOKEN_EXPIRATION_TIME=1337s
```

Comment:
I would like to suggest add postfix of time measurement, e.g. `BLAH_BLAH_SECONDS/MINUTES/HOURS`